homework/03LessonThree.js

```diff
@@ -89,17 +88,18 @@ var problemSevenAnswer;
 
 // Problem 8:
 // Create an empty array
-var emptyArray;
+var emptyArray = [  ];
 
 // Problem 9:
 // Create an array with 5 elements in it
-var lengthFiveArray;
+var lengthFiveArray = [ 1, 2, 3, 4, 5 ];
 
 // DO NOT MODIFY
 var nameArray = ['bob', 'fred', 'susan'];
 
 // Problem 10:
 // Replace the value 'fred' in nameArray with 'george'
+    var nameArray = [ 'bob', 'george', 'susan' ]
```

Comment:
Technically right, but we want you to use the indexing syntax to change the original `nameArray`.
So try this one again without redeclaring nameArray